src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,111 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement createStatement =
+                         connection.prepareStatement(
+                             "INSERT INTO manufacturers(name, country) values(?,?);",
+                             Statement.RETURN_GENERATED_KEYS)) {
+            createStatement.setString(1, manufacturer.getName());
+            createStatement.setString(2, manufacturer.getCountry());
+            ResultSet generatedKeys = createStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                Long id = generatedKeys.getLong(1);
+                manufacturer.setId(id);
+            }
+            createStatement.executeUpdate();
+        } catch (SQLException e) {
+            throw new RuntimeException("Error");
+        }
+        return manufacturer;
+    }
+
+    @Override
+    public Manufacturer get(Long id) {
```

Comment:
As commented in the interface this method should return Optional of Manufacturer when the result is found or Optional.empty() if there is no result.
Additionally,  when you're retrieving id of the returned object, better practice is to use getObject, which will return null value if value in database is absent. In your implementation it will return 0, which may be confusing and suggesting that object was created with id 0. 
For better readability, I'd move manufacturer creation to separate private method

As commented under get method implementation - you should throw custom exception and provide meaningful message. 

```suggestion
  @Override
  public Optional<Manufacturer> get(Long id) {
    String getManufacturerRequest = "SELECT * FROM manufacturers WHERE id = ?";
    try (Connection connection = ConnectionUtil.getConnection();
        PreparedStatement getStatement = connection.prepareStatement(getManufacturerRequest)) {
      getStatement.setLong(1, id);
      ResultSet resultSet = getStatement.executeQuery();
      if (resultSet.next()) {
        // return Optional of manufacturer if found
        return Optional.of(mapManufacturer(resultSet));
      } else {
        // return optional empty if not found
        return Optional.empty();
      }
    } catch (SQLException e) {
      throw new DataProcessingException("There was an error getting manufacturer for id: " + id);
    }
  }

  private Manufacturer mapManufacturer(ResultSet resultSet) throws SQLException {
    // use getObject instead of getLong
    Long idOfManufacturer = resultSet.getObject("id", Long.class);
    String name = resultSet.getString("name");
    String country = resultSet.getString("country");
    return new Manufacturer(idOfManufacturer, name, country);
  }

```